mirage/config.js

```diff
@@ -11,6 +11,7 @@ export default function () {
   this.passthrough('https://raw.githubusercontent.com/**');
   this.passthrough('https://tiles.planninglabs.nyc/**');
   this.passthrough('https://layers-api-staging.planninglabs.nyc/**');
+  this.passthrough('https://labs-layers-api-staging.herokuapp.com/v1/base/style.json');
```

Comment:
What's the difference between `https://layers-api.planninglabs.nyc/v1/base/style.json` and `https://labs-layers-api-staging.herokuapp.com/v1/base/style.json`? I noticed CEQR uses the first one.

Reply:
Not sure, I didn't make the original changes that affected this, but having both of them there is necessary to get tests to pass

Reply:
Ok! 👌